### ISSUE TITLE
(Question) Cuda 11.1 vs 11.2 vs 11.3 vs 11.4

### ISSUE THREAD

**Issue by kiatchun** [lightvector/KataGo#568]
Is there a stability/performance difference between using Cuda 11.1 vs 11.2 vs 11.3 vs 11.4?

**Comment by kennychenfs** [comment]
I think newer version is better.

**Comment by lightvector** [comment]
Yes there should be some differences, newer versions may be better so long as they are compatible, but the difference is not so large.

KataGo has been tested most with 11.1 and 11.2, but up through 11.4 should be fine. 

I would *not* rush to switch to even newer versions as they are released, because usually there is very little performance difference, and sometimes newer versions of CUDA introduce performance regressions or bugs, so it's can be worth waiting a bit for a CUDA release to be out for a while and become more tested and stable before using it. You can see that there is often an "update 1" or "update 2" for each CUDA version fixing such bugs.

**Comment by kiatchun** [comment]
Is there a difference between cuda version 11.x.1 and 11.x update 1?
